frontend/ui/src/components/IconChannelFilter/index.tsx

```diff
@@ -6,12 +6,14 @@ import {ReactComponent as GoogleIcon} from 'assets/images/icons/google_avatar.sv
 import {ReactComponent as WhatsappIcon} from 'assets/images/icons/whatsapp_avatar.svg';
 import {ReactComponent as SmsIcon} from 'assets/images/icons/sms_avatar.svg';
 import {ReactComponent as FacebookIcon} from 'assets/images/icons/messenger_avatar.svg';
+import {ReactComponent as ChatPluginSource} from 'assets/images/icons/airy_avatar.svg';
```

Comment:
AiryAvatar? We should start at some point :)

Reply:
true dat